api/model/api.go

```diff
@@ -1163,6 +1163,13 @@ type GraphQLConfig struct {
 
 	// Supergraph holds the configuration for a GraphQL federation supergraph.
 	Supergraph GraphQLSupergraphConfig `json:"supergraph,omitempty"`
+
+	// Introspection holds the configuration for GraphQL Introspection
+	Introspection GraphQLIntrospectionConfig `json:"introspection,omitempty"`
+}
+
+type GraphQLIntrospectionConfig struct {
+	Disabled bool `json:"disabled,omitempty"`
```

Comment:
Can we set default value to true?

Reply:
@komalsukhani  on second thought, I propose setting this value to false by default, as it aligns with users' expectations to see the result of GraphQL introspection by default. For those who prefer an alternative to disable introspection, a configuration option is available on Tyk side to disable it. Opting for a default of true might potentially lead to confusion. What are your thoughts on this approach?

Reply:
Won't it cause problem as API Definition will not match with CRD?

Reply:
I do not think that as APIDefinition on Tyk is only compared with the hash of ApiDefinition on Tyk that is obtained after creating CRD.

Reply:
okay. We can confirm that during testing.